src/business-rules/claim-payout-amount.ts

```diff
@@ -44,3 +46,26 @@ export const isUnrepairableOrTooExpensive = (isRepairable, repairCostIsTooHigh)
   
   return repairCostIsTooHigh
 }
+
+const computePayout = (...values) => {
+  const filteredValues = [...values]?.filter(value => value !== undefined)
+  return formatMoney(Math.min(...filteredValues) * (1 - deductible))
+}
+
+const computeRepairMaxPayout = (claimItem: ClaimItem, coverageAmount) => computePayout(claimItem.repair_actual || claimItem.repair_estimate, coverageAmount, claimItem.fmv)
+
+const computeReplaceMaxPayout = (claimItem: ClaimItem, coverageAmount) => computePayout(claimItem.replace_estimate, coverageAmount)
+
+const computeCashMaxPayout = (claimItem: ClaimItem, coverageAmount) => computePayout(coverageAmount, claimItem.fmv)
+
+export const determineMaxPayout = (payoutOption, claimItem: ClaimItem, coverageAmount) => {
+  if(payoutOption === PAYOUT_OPTION_REPAIR) {
+    return computeRepairMaxPayout(claimItem, coverageAmount)
+  } else if(payoutOption === PAYOUT_OPTION_REPLACE) {
+    return computeReplaceMaxPayout(claimItem, coverageAmount)
+  } else if(payoutOption === PAYOUT_OPTION_FMV) {
+    return computeCashMaxPayout(claimItem, coverageAmount)
+  } else if(payoutOption === PAYOUT_OPTION_FIXED_FRACTION) {
+    return formatMoney(coverageAmount * 2/3) || ''
+  }
```

Comment:
This is totally just a suggestion... I mostly wanted to see how it would look if we did it this way.

I tend to find multiple if/else-if/else's chained together to be somewhat hard to read. If you're interested, this is a way to do it as an object, so there is less boiler-plate code.:
```suggestion
  const calculationsByPayoutOption = {
    PAYOUT_OPTION_REPAIR: () => computeRepairMaxPayout(claimItem, coverageAmount),
    PAYOUT_OPTION_REPLACE: () => computeReplaceMaxPayout(claimItem, coverageAmount),
    PAYOUT_OPTION_FMV: () => computeCashMaxPayout(claimItem, coverageAmount),
    PAYOUT_OPTION_FIXED_FRACTION: () => formatMoney(coverageAmount * 2/3) || '',
  }
  return calculationsByPayoutOption[payoutOption]()
```

You are more than welcome to keep your code as-is, I'm just tossing this out there as another option.